cypress/plugins/index.js

```diff
@@ -16,19 +16,34 @@ module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
 
-  // on('before:browser:launch', (browser = {}, args) => {
-  //   if (browser.name === 'chrome') {
-  //     args.push('--disable-blink-features=RootLayerScrolling')
-  //     return args
-  //   }
-  // })
+  on('before:browser:launch', (browser = {}, args) => {
+    //require('../GenerateCypressTestSpecs')
+    
+    // BUG BUG - Need to work out why this code does not execute or where it is throwing an exception.
+    alert('got here');
+    var fs = require('../../node_modules/pn/fs')
```

Comment:
This path looks strange. Why did you use this one? 

Normally for global modules you only specify the module name like:
`var fs = require('fs')` OR better `import * as fs from 'fs'`

Reply:
That code still needs to be fixed, but it does not appear to be influencing the test runs in anyway so I left it in for the next round of work on it. 

However, when I did have something working in this plugin file I had to do the require that way, I am not sure why, but it was the only way that worked. There are many things that Cypress is doing that interferes with "normal" JavaScript or Node coding.

Reply:
I commented out that code for now as well.

Reply:
Ah, I just figured out the issue with this file and the troubles I've had with it. You must completely close and restart Cypress for changes in this file to take affect. Which explains why the require gave me troubles originally. I have fixed it.